notebooks/official/ray_on_vertex/ray_cluster_management.ipynb

```diff
@@ -0,0 +1,638 @@
+{
```

Comment:
<ul><li>how to creating cluster, listing clusters, getting a cluster, updating (manually scaling) cluster, and deleting cluster. &gt; how to create a cluster, get a cluster, update (manually scale) a cluster, and delete a cluster</li><li>Ray on Vertex &gt; Ray on Vertex AI</li><li>Confirm if the notebook is using Vertex AI Training?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Good call! I removed Vertex AI Training and keep Ray on Vertex AI.